### ISSUE TITLE
EIS "e4" not work on Hero12 & MLM 2.0

### ISSUE THREAD

**Issue by euromarco** [gopro/labs#1045]
ciao ragazzi, 
I couldn't get the Autoboost to activate on EIS (Hypersmooth control) with lens Max 2.0
it doesn't take the command "e4" in whatever position I tried to insert it.
this is my string: mSr4mSoX2fXLp30 [NB repetition of the "mS" command is necessary]
Has anyone managed to modify the Hypersmooth control successfully? Thank you,
Marco

**Comment by dnewman-gpsw** [comment]
Correct.  e4 Is not currently working with Max Lens 2.0.  This is a bug.  You will have to use the menu or Quik.

**Comment by euromarco** [comment]
Grazie

**Comment by euromarco** [comment]
Hi, still any update on Autoboost functionality on EIS (Hypersmooth control) with Max 2.0 lens?

**Comment by dnewman-gpsw** [comment]
It was not fixed.

**Comment by dnewman-gpsw** [comment]
I did just discover the error. No builds currently planned for HERO12, but the fix will be in the next HERO13 Labs release.
